src/components/Editable.tsx

```diff
@@ -439,7 +440,7 @@ const Editable = ({ disabled, isEditing, simplePath, path, cursorOffset, showCon
     const oldValueClean = oldValue === EM_TOKEN ? 'em' : ellipsize(oldValue)
 
     const thoughtsInContext = getAllChildren(state, context)
-    const hasDuplicate = newValue !== '' && thoughtsInContext.some(thought => thought.value === newValue)
+    const hasDuplicate = newValue !== '' && thoughtsInContext.some(thought => normalizeThought(thought.value) === normalizeThought(newValue))
```

Comment:
You’ll want to normalize `newValue` outside the loop for efficiency.

Reply:
Sure